ReactNativeClient/lib/components/screens/encryption-config.js

```diff
@@ -150,6 +155,16 @@ class EncryptionConfigScreenComponent extends BaseScreenComponent {
 						this.setState({ passwordPromptAnswer: text });
 					}}
 				></TextInput>
+				<Text>Confirm Password: </Text>
+				<TextInput
+					selectionColor={theme.textSelectionColor}
+					style={{ margin: 10, color: theme.color, borderWidth: 1, borderColor: theme.dividerColor }}
```

Comment:
Please move the style to a variable so that it can be reused by this component and the previous one.